src/libraries/Microsoft.PowerFx.Connectors/OpenApiExtensions.cs

```diff
@@ -766,8 +765,21 @@ private static ConnectorType TryGetOptionSet(ISwaggerParameter openApiParameter,
                     }
                     else if (schema.Enum.All(e => e is OpenApiInteger))
                     {
-                        // $$$ Not supported yet
-                        return new ConnectorType("Unsupported enum type (int)", openApiParameter.Name, FormulaType.BindingError);
+                        var enumName = openApiParameter.Name;
+#pragma warning disable CA1305 // Specify IFormatProvider

```

Comment:
Can we eliminate this warning and speciy the formatter? Is this CultureInfo.InvariantCulture?